vllm/model_executor/layers/mamba/ops/ssd_state_passing.py

```diff
@@ -27,60 +27,42 @@ def _state_passing_fwd_kernel(
     # Pointers to matrices
     states_ptr,
     out_ptr,
-    final_states_ptr,
     dA_cs_ptr,
     initstates_ptr,
     seq_idx_ptr,
     # Matrix dimensions
-    dim,
+    dim: tl.constexpr,
     nchunks,
     seqlen,
-    chunk_size,
+    chunk_size: tl.constexpr,
     # Strides
-    stride_states_batch,
-    stride_states_chunk,
-    stride_states_head,
-    stride_states_dim,
-    stride_out_batch,
-    stride_out_chunk,
-    stride_out_head,
-    stride_out_dim,
-    stride_final_states_batch,
-    stride_final_states_head,
-    stride_final_states_dim,
-    stride_dA_cs_batch,
-    stride_dA_cs_chunk,
-    stride_dA_cs_head,
-    stride_initstates_batch,
-    stride_initstates_head,
-    stride_initstates_dim,
-    stride_seq_idx_batch,
-    stride_seq_idx_seqlen,
+    stride_states_chunk: tl.int64,
+    stride_states_head: tl.int64,
+    stride_states_dim: tl.int64,
+    stride_out_chunk: tl.int64,
+    stride_out_head: tl.int64,
+    stride_out_dim: tl.int64,
+    stride_dA_cs_head: tl.int64,
+    stride_dA_cs_chunk: tl.int64,
+    stride_initstates_batch: tl.int64,
+    stride_initstates_head: tl.int64,
+    stride_initstates_dim: tl.int64,
+    stride_seq_idx_seqlen: tl.int64,
```

Comment:
Sometimes we mark the strides all as `tl.constexpr`, sometimes all of them as `tl.int64` and sometimes a mixture. Should we try to be consistent?

Reply:
yeah, I'll go through them again. The convention I'll use is: **tl.int64 for strides, except for the least significant dimension, which is set to tl.constexpr (it's always 1)**